src/translations/IntlContext.tsx

```diff
@@ -0,0 +1,57 @@
+import React, { useState, useContext } from "react";
+import { IntlProvider } from "react-intl";
+import sk from '../compiled-lang/sk';
+import en from '../compiled-lang/en';
+
+const Context = React.createContext(null);
+
+const IntlConsumer = (props) => {
+  const loadLocaleData = locale => {
```

Comment:
Let's:
- get rid of this function
- we will have array of objects like this: [ { locale, messages } ]
- first item in the array will be EN and it will be default